pom.xml

```diff
@@ -102,6 +103,22 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <requireUpperBoundDeps>
+              <excludes combine.children="append">
+                <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
+                <exclude>org.apache.ant:ant</exclude>
```

Comment:
I need to learn how to ignore a dependency trail using `<exclusion>`.  Any pointers to a place where I can see that technique?